terraform/aptos-node-testnet/aws/addons.tf

```diff
@@ -256,6 +256,9 @@ resource "helm_release" "testnet-addons" {
         loadBalancerSourceRanges = var.client_sources_ipv4
       }
       load_test = {
+        image = {
+          repo = "323143817725.dkr.ecr.us-west-2.amazonaws.com/aptos/txn-emitter"
```

Comment:
this is only for our internal repo, don't include in this PR

Reply:
Fixed.